scripts/lib/setup_venv.py

```diff
@@ -45,19 +45,16 @@
     "jq",                   # Used by scripts/lib/install-node to check yarn version
 ]
 
-YUM_VENV_DEPENDENCIES = [
+COMMON_YUM_VENV_DEPENDENCIES = [
     "libffi-devel",
     "freetype-devel",
     "zlib-devel",
     "libjpeg-turbo-devel",
     "openldap-devel",
     "libmemcached-devel",
-    "python34-devel",
     "python-devel",
-    "python34-pip",
     "python2-pip",
     "python-virtualenv",
-    "python34-six",
     "python-six",
     "libxml2-devel",
```

Comment:
You're going to regret this when you get to working on production, since this value is used for installation there as well.

As I said, we should just have per-platform values here.

Reply:
The per-platform values are constructed on the spot at provision.py. Did you mean to move them to here?

Reply:
Right, they should be constructed here, so that we can reuse the logic in prod.